rastervision_pytorch_learner/rastervision/pytorch_learner/dataset/transform.py

```diff
@@ -23,10 +23,11 @@ def classification_transformer(inp: Tuple[Any, Any],
                                ) -> Tuple[np.ndarray, np.ndarray]:
     """Apply transform to image only."""
     x, y = inp
-    x, y = np.array(x), np.array(y, dtype=np.long)
+    x, y = np.array(x), np.array(y)
     if transform is not None:
         out = transform(image=x)
         x = out['image']
+    y = y.astype(np.long)
```

Comment:
This is not necessary for classification (because `y` is not modified by the transform), but moving the casting to the end emphasizes that the returned `y` should be a `long`.